terraform/main.tf

```diff
@@ -20,12 +20,7 @@ terraform {
 
   required_version = ">= 1.5.7"
 
-  backend "s3" {
-    bucket = "ccao-terraform-state-us-east-1"
-    key    = "terraform.tfstate"
-    region = "us-east-1"
-    workspace_key_prefix = "model-res-avm/workspaces"
-  }
+  backend "s3" {}
```

Comment:
**question:** So just to clarify, these variables are now set via the `-backend-config` flags because we want them to be set for different for each repo?

Reply:
That's right! I added a comment explaining this in [963eda5](https://github.com/ccao-data/model-res-avm/pull/58/commits/963eda54e1420861ff007430651fe158ed3485a5).